util/diff_runs.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python3
```

Comment:
Sadly we're using python 2 at the moment because WPT infra is all python 2. See https://github.com/w3c/wptdashboard/pull/121

Also: :sob: :sob: :sob: :sob: :sob: